### DISCUSSION TITLE
minio zabbix template

### DISCUSSION THREAD

**Discussion by Karotin666** [minio/minio#19943] [Q&A]
Hi.
Does anyone have zabbix templates for minio?

**Comment by bebianc** [comment]
Hi @Karotin666 
What version of MinIO are you using?

**Reply by GhaziTriki** [reply]
@Karotin666 Check this please it working very well https://github.com/KjellWolf/CW-Zabbix-Templates

**Comment by rob4226** [comment]
I also found this one:
https://gitlab.com/eliwinderickx/minio-monitoring-template

**Reply by aozima** [reply]
very easy to use👍
